maestro-android/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,29 @@
                      tools:ignore="MockLocation,ProtectedPermissions"/>
     <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES"
                      tools:ignore="QueryAllPackagesPermission"/>
+    <uses-permission android:name="android.permission.WRITE_SETTINGS"
+                     tools:ignore="ProtectedPermissions" />
+    <uses-permission android:name="android.permission.CHANGE_CONFIGURATION"
+                     tools:ignore="ProtectedPermissions" />
 
-    <application android:usesCleartextTraffic="true" />
+    <application
+            android:usesCleartextTraffic="true">
+            <activity
```

Comment:
Now that we're adding a `MainActivity` to the runner app, might this have any other side effects?

Reply:
generally every Android app has a main activity, so I don't see any bad side-effects happening by adding it to maestro driver app. cc @amanjeetsingh150

Reply:
Yep unsure here as well, but given that we're changing internals of how maestro works I want us to be extra careful with testing so we don't break anything. Could you share more on what the plan is to verify that this is a safe change?

Reply:
checked, it works without an Activity, thanks for finding 👁️